README.md

```diff
@@ -2,18 +2,265 @@
 
 ## Getting Started
 
-First, run the development server:
+### Installation
+
+1. On linux debian based systems, run to install npm
+
+```bash
+sudo apt install npm
+```
+
+**Note: There may be other dependancies that need installed with nmp**
+
+2. Clone the repository to your local environment
+
+```bash
+git clone https://github.com/esilverm/shl-index-ui
+```
+
+**Note: when making changes, it is best to fork the repository and submit pull requests**
+
+3. Navigate to the directory
+
+```bash
+cd shl-index-ui
+```
+
+4. Install all libraries
+
+```bash
+npm install
```

Comment:
Im going to prefer that we use yarn instead

Reply:
that is fine.  I will adjust the content and let you know. I just learned npm for this... so now I get to learn yarn!

Reply:
great! thanks! yarn is a lot faster and is much better at caching dependencies than npm so I tend to prefer it. the difference is very minute. instead of `npm install [package name]` you run `yarn add [package name]`. Also to run the dev server instead of telling npm to `run` the command from your `package.json`, you just do `yarn` and then the command name.

Reply:
should be updated @esilverm